.github/home.css

```
@@ -249,4 +257,246 @@ footer {
+    margin: 0;
+  }
+
+  .nav {
+    margin: 0;
+    padding: 0;
+  }
+
+  .nav a {
+    margin: 0;
+  }
+
+  .nav > a > img {
+    width: 133px;
+    height: 24px;
+  }
+
+  .login_bt {
+    width: 128px;
+    padding: 16px 20px;
+    gap: 10px;
+  }
+
+  header {
+    display: flex;
+    width: 1199px;
+    padding: 100px 250px 0;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    gap: 40px;
+  }
+
+  header > h1 {
+    height: fit-content;
+    margin: 0;
+    font-size: 56px;
+    text-align: center;
+  }
+
+  header > a {
+    display: flex;
+    width: 350px;
+    padding: 16px 20px;
+    justify-content: center;
+    align-items: center;
+    gap: 10px;
+  }
+
+  .header_img {
+    width: 698px;
+    height: 390px;
+    justify-content: center;
+  }
+  
+  .header_img > img {
+    width: 698px;
+  }
+
+  section {
+    height: 440px;
+    padding: 80px 250px 50px;
+  }
+  
+  .section_contents {
+    width: 698px;
+    height: 315px;
+    gap: 51px;
+  }
+  
+  .section_text {
+    width: 262px;
+  }
+  
+  
+  .section_text>h2 {
+    font-size: 2.5rem;
+  }
+  
+  .section_text>p {
+    font-size: 13.5px;
+  }
+  
+  .section_contents > div > img {
+    width: 385px;
+  }
+  
+  .footer_contents {
+    width: 991px;
+  }
+}
+
+@media screen and (min-width: 375px) and (max-width: 767px) {
+  * {
+    max-width: 767px;
+  }
+
+  nav {
+    padding: 13px 32px;
+  }
+
+  .nav {
+    padding: 0;
+  }
+
+  .login_bt {
+    width: 80px;
+    height: 20px;
+    margin: 0;
+    font-size: 12px;
+  }
+
+  .nav > a > img {
+    width: 89px;
+    margin: 0;
+  }
+
+  .nav a {
+    margin: 0;
+  }
+
+  header {
+    width: 767px;
+    padding: 28px 32px 0px;
+    gap: 24px;
+  }
+
+  header > h1 {
+    height: fit-content;
+    margin: 100px 0 0 0;
+    font-size: 36px;
+    line-height: 42px;
+  }
+
+  header > a {
+    width: 200px;
+    height: 40px;
+    padding: 10px 16px;
+    font-size: 15px;
+  }
+
+  .header_img {
+    height: 340px;
+    padding: 0;
+  }
+  
+  .header_img > img {
+    width: 700px;
+    margin: 0 32px;
+  }
+
+  section {
+    width: 767px;
+    height: fit-content;
+    padding: 40px 0 36px;
+    justify-content: center;
+  }
+  
+  .section_contents {
+    width: 767px;
+    height: fit-content;
+    margin: 0;
+    flex-direction: column;
+    justify-content: center;
+    gap: 24px;
+  }
+  
+  .reverse {
+    flex-direction: column-reverse;
+  }
+
+  .section_text {
+    width: 400px;
+    align-items: center;
+    gap: 10px;
+  }
+  
+  .section_text>h2 {
+    font-size: 30px;
+  }
+  
+  .section_text>p {
+    width: 400px;
+    font-size: 15.5px;
+  }
+
+  .section_contents > div {
+    width: 767px;
+  }
+  
+  .section_contents > div > img {
+    width: calc(100% - 64px);
+    height: auto;
+    margin: 0 32px;
+  }
+
+  footer {
+    height: 280px;
+  }
+  
+  .foot {
+    padding: 40px 32px 0;
+  }
+
+  .footer_contents {
+    display: grid;
+    padding: 0 50px 0 0;
```

```suggestion
    padding-right:50px;
```
여기도 마찬가지로 특정 영역에만 주고 싶으면 특정 영역을 지정해서 적용해주세요!